system_tests/bigquery.py

```diff
@@ -29,6 +29,11 @@
 
 DATASET_NAME = 'system_tests' + unique_resource_id()
 
+# We need to wait to stay within the rate limits.
+# The alternative outcome is a 403 Forbidden response from upstream.
+# See: https://cloud.google.com/bigquery/quota-policy
```

Comment:
@tseaver Can you add an extra note that they (strangely) use 403 **instead of** "429 Too Many Requests" for rate-limiting? Just for future people who might think "whaaaah"?